src/utils.jl

```diff
@@ -109,14 +109,18 @@ function compute_theory(DL_TT, DL_TE, DL_EE, κ,#1
                                        ν_eff[2,3], ν_eff[2,3], SPT3G_windows_lmax)) ./
                                     _calibration(cal_T_220, cal_E_220, cal_T_220, cal_E_220)
 
-    pure_theory = hcat(TT_90_90, TE_90_90, EE_90_90,
-                       TT_90_150, TE_90_150, EE_90_150,
-                       TT_90_220, TE_90_220, EE_90_220,
-                       TT_150_150, TE_150_150, EE_150_150,
-                       TT_150_220, TE_150_220, EE_150_220,
-                       TT_220_220, TE_220_220, EE_220_220)
-
-    model_matrix = hcat([@views window[:,i,:]*pure_theory[:,i] for i in 1:18]...)'
+    pure_theory = [
+      TT_90_90, TE_90_90, EE_90_90,
+      TT_90_150, TE_90_150, EE_90_150,
+      TT_90_220, TE_90_220, EE_90_220,
+      TT_150_150, TE_150_150, EE_150_150,
+      TT_150_220, TE_150_220, EE_150_220,
+      TT_220_220, TE_220_220, EE_220_220
+    ]
+
+    model_matrix = stack(1:18) do i
+       @view(window[:,:,i])' * pure_theory[i]
```

Comment:
In Julia you want the "batch" dimension last (unlike Python where you want it first). Also `A' * x` is faster than `A * x` because the former goes through the memory of `A` in row order. See also corresponding `permutedim` when loading `windows`.